server/index.js

```diff
@@ -8,28 +8,131 @@ const path = require('path');
 const { QueryTypes } = require('sequelize');
 const sequelize = require('./database/index.js');
 const cors = require('cors');
+const bodyParser = require('body-parser');
 
 app.use(cors());
-
+app.use(bodyParser());
 app.use(express.static(path.join(__dirname, '../client/public/dist')))
 
 
+// ------------------------------------------------------- //
+// ------------- SDC ADDITIONS TO CODEBASE --------------- //
+// ------------------------------------------------------- //
+
+// to display content based on product ID in URL path e.g. localhost:3002/4
+app.use('/:productId', express.static(path.join(__dirname, '../client/public/dist')))
+
+// helper functions
+const findProduct = async function(prodId) {
+  const productData = await db.Product.findAll({
+    where: {
+      id: prodId
+    },
+    attributes: {exclude: ['createdAt', 'updatedAt']}
+  })
+  return productData
+}
+
+const updateProduct = function(prodId, prodInfo) {
+  db.Product.update({
+    name: prodInfo.name,
+    price: prodInfo.price,
+    reviews: prodInfo.reviews,
+    reviewCount: prodInfo.reviewCount
+  }, {
+    where: {
+      id: prodId
+    }
+  });
+}
+
+const deleteProduct = function(prodId) {
+  db.Product.destroy({
+    where: {
+      id: prodId
+    }
+  });
+}
+
+
+// CREATE
+app.post('/api/products', async (req, res) => {
+  try {
+    db.Product.create({
+      name: req.body.name,
+      price: req.body.price,
+      reviews: req.body.reviews,
+      reviewCount: req.body.reviewCount
+    })
+    res.send('success')
+  } catch (e) {
+    console.error(e)
```

Comment:
I think you may want to do more than just console.error. Otherwise the response won't get sent and the the request will time out without the client knowing anything about what happened.

I'm still personally trying to figure out what the proper response is, so I don't necessarily have the answer.